package.json

```diff
@@ -7,8 +7,10 @@
     "start": "swagger project start",
```

Comment:
seems like `swagger` should be a dependency?

Reply:
Are the Swagger modules providing any value? It seems like we have the Swagger docs in a file, right?

Reply:
@nonword the original README suggested installing Swagger globally, `npm install swagger -g` but I'll change that and add it as a dependency.

@kfriedman the README suggests that we can edit the documentation with interactive UI, after running this command: `npm run swagger-edit`.  So that seems that there is some value.  Are you suggesting that we should or should not include Swagger as a dependency?

Reply:
Yes, I guess I was wondering whether it should be a dependency. At the very least, I don't think it should be a production dependency. 

Also, I noticed `pg-promise` is a dependency. We don't use PostgreSQL anywhere, do we? 

It might be worth reviewing to make sure we're actually using all the dependencies.

Reply:
True. Should probably be a dev-dependency. Seems like a handy util for running a server locally (and necessary for integration tests), but given that it's deployed as a lambda, no need for it as a production dependency. Lately I think we require all dependencies to be explicit, so the README suggestion that one install swagger globally is, I think, outdated.

Reply:
@nonword ready for another look. Thank you!